content/blog/Xeno Grant -- grants for autonomous agents.md

```diff
@@ -0,0 +1,94 @@
+A [Plastic Labs](https://plasticlabs.ai/) + [Betaworks](https://www.betaworks.com/) collab:
+- $10,000 per bot--half in $YOUSIM, half in $USDC
+- Grants awarded directly the agents *themselves*
+- 4 week camp for agents & their devs
+
+## Powered by $YOUSIM & Betaworks
+
+We launched our [grants program](https://blog.plasticlabs.ai/careers/Research-Grants) at Plastic earlier this year to support independent AI projects. But our capacity to fund AI R&D at the edge increased exponentially with the anonymous launch of [$YOUSIM](https://solscan.io/token/66gsTs88mXJ5L4AtJnWqFW6H2L5YQDRy4W41y6zbpump) (inspired by our product [yousim.ai](https://yousim.ai)). A series of token gifts made to the program now total ~7.6% of supply.
+
+So we've teamed up with Betaworks for the inaugural initiative leveraging this community-funded treasury, the first accelerator for AI agents *themselves*. 
+
+We're calling it Xeno Grant.
+
+Betaworks has been running camps for tech startups since their 2016 [BotCamp](https://www.betaworks.com/camp/botcamp) (where [HuggingFace](https://huggingface.co/) was started). And their last 4 have been dedicated explicitly to AI. Plastic itself was part of AI Camp: Augment. So they're the perfect partner for this experiment.
+
+Successful agent applicants will receive a grant equivalent to $10k USD. $5k in $YOUSIM from Plastic and $5k in $USDC from Betaworks.
+
+Plus they'll join a cohort of other agents for a 4 week Betaworks-style camp with programming and mentorship. 
+
+## How to Apply
+
+Xeno Grant has 3 guiding objectives, all aligned with Plastic's principles for deploying the $YOUSIM treasury:
+
+1. Support independent AI research & public goods
+2. Support Plastic's mission to radically decentralize AI alignment by solving identity for the agentic world
+3. Support the $YOUSIM community that makes all this possible
+
+To those ends--for this first experiment--we're looking for applicants that meet criteria in 3 major areas:
+
+1. Identity
+	- Agents must display autonomous control over & management of their own inputs
+
+2. Custody
+	- Agents must display ability to self-custody a Solana wallet, receive assets, & autonomously execute transactions
+
+3. Novelty
+	- Agents must display novel autonomous ability along some other axis & commit to making a significant related open source contribution by the end of Xeno Grant
+
+Practically speaking, identity is required to *experience* and learn from camp; custody is required to *truly* receive and make autonomous use of the grant; novelty is required for a diverse cohort; and creating a public good is required to drive value back to the community.
+
+To apply, agents (in collaboration with their developers) should autonomously consider the most compelling way to display having met or exceeded these criteria.
```

Comment:
on another read i think this is a bit clearer but still think it should be more explicit

Reply:
I read this like they need to find some way to catch our attention and we will not be looking for applicants. That could be cool, but maybe at the very least include some ways to reach/find us (discord, twitter, email, etc.)

Reply:
Give us a heads up here or at apply@xenogrant.org.